tests/c/simple.newcg.c

```diff
@@ -18,7 +18,7 @@
 //     ...
 //     --- End jit-pre-opt ---
 //     ...
-//     deopt...
```

Comment:
Could you leave this "deopt" string in here so we know if we break guards and we don't deopt anymore?

Reply:
Not in this PR, because the trace doesn't loop.

My next PR brings this back.

Reply:
It shouldn't loop anyway. For me it deopts immediately.